ui/public/scripts/controllers/mobileoverview.js

```diff
@@ -39,23 +43,71 @@ angular.module('mobileControlPanelApp').controller('MobileOverviewController', [
         $scope.project = project;
         $scope.projectContext = projectContext;
 
-        DataService.list(
-          {
-            group: 'servicecatalog.k8s.io',
-            resource: 'serviceclasses'
-          },
-          $scope.projectContext,
-          function(serviceClasses) {
-            $scope.serviceClasses = serviceClasses._data;
-          }
-        );
+        $scope.overviews.apps = {
+          title: 'Mobile Apps',
+          actions: [
+            {
+              label: 'Create Mobile App',
+              primary: true,
+              action: $location.path.bind(
+                $location,
+                `project/${projectContext.projectName}/create-mobileapp`
+              ),
+              canView: function() {
+                return true;
+              }
+            }
+          ]
+        };
+        $scope.overviews.services = {
+          title: 'Mobile Enabled Services',
+          actions: [
+            {
+              label: 'Add External Service',
+              action: $location.path.bind(
+                $location,
+                `project/${projectContext.projectName}/create-mobileservice`
+              ),
+              canView: function() {
+                return AuthorizationService.canI(
+                  'services',
+                  'create',
+                  projectContext.projectName
+                );
+              }
+            },
+            {
+              label: 'Provision Catalogue Service',
```

Comment:
better to go with the American spelling that's used elsewhere in OpenShift -> "Catalog"

Reply:
Yep